Description: Apply patch to fix test-suite changes with new git.
Author: Dimitri John Ledkov <xnox@ubuntu.com>

Index: guilt-0.36/regression/t-020.out
===================================================================
--- guilt-0.36.orig/regression/t-020.out
+++ guilt-0.36/regression/t-020.out
@@ -228,7 +228,6 @@ Invalid number of patches to push.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -243,7 +242,6 @@ Invalid number of patches to push.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -258,7 +256,6 @@ Patch is already applied.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -357,7 +354,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -478,7 +474,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 0803c721968056410df61400bb239380d033b9d5  .git/patches/master/empty.patch~
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
Index: guilt-0.36/regression/t-021.out
===================================================================
--- guilt-0.36.orig/regression/t-021.out
+++ guilt-0.36/regression/t-021.out
@@ -74,7 +74,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -158,7 +157,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -225,7 +223,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -364,7 +361,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
Index: guilt-0.36/regression/t-025.out
===================================================================
--- guilt-0.36.orig/regression/t-025.out
+++ guilt-0.36/regression/t-025.out
@@ -125,8 +125,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -149,8 +147,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -172,8 +168,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -195,8 +189,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -219,8 +211,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -243,8 +233,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -267,8 +255,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -291,8 +277,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -315,8 +299,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -339,8 +321,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -363,8 +343,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -387,8 +365,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -411,8 +387,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -435,8 +409,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -459,8 +431,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -483,8 +453,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -507,8 +475,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -531,8 +497,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -555,8 +519,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -579,8 +541,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -603,8 +563,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -627,8 +585,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -651,8 +607,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -675,8 +629,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -699,8 +651,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -723,8 +673,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -747,8 +695,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -771,8 +717,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -795,8 +739,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -836,8 +778,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 1ef04b4f4149870b3f06b5f99b9d9fc260cebc67  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -864,8 +804,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 06d2f9f2f2c01d41544a0eab8040959c318e8c3b  .git/patches/master/uncommitted-changes
 f 0d14139c7f15f095b9a3dae9802f20072a4c1859  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
@@ -914,8 +852,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 06d2f9f2f2c01d41544a0eab8040959c318e8c3b  .git/patches/master/uncommitted-changes
 f 0d14139c7f15f095b9a3dae9802f20072a4c1859  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
@@ -945,8 +881,6 @@ d .git/patches/master/dir
 d .git/patches/master/dir/subdir
 d .git/refs/patches
 d .git/refs/patches/master
-d .git/refs/patches/master/dir
-d .git/refs/patches/master/dir/subdir
 f 06d2f9f2f2c01d41544a0eab8040959c318e8c3b  .git/patches/master/uncommitted-changes
 f 0fe110e08d3278d1c5757ee600c0a6c043f0cc07  .git/patches/master/status
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
Index: guilt-0.36/regression/t-026.out
===================================================================
--- guilt-0.36.orig/regression/t-026.out
+++ guilt-0.36/regression/t-026.out
@@ -37,7 +37,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 7848194fd2e9ee0eb6589482900687d799d60a12  .git/patches/master/series
Index: guilt-0.36/regression/t-029.out
===================================================================
--- guilt-0.36.orig/regression/t-029.out
+++ guilt-0.36/regression/t-029.out
@@ -93,7 +93,6 @@ Repair complete.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -135,7 +134,6 @@ Repair complete.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
Index: guilt-0.36/regression/t-030.out
===================================================================
--- guilt-0.36.orig/regression/t-030.out
+++ guilt-0.36/regression/t-030.out
@@ -112,7 +112,6 @@ r ffb7faa126a6d91bcdd44a494f76b96dd860b8
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 3516b9a25ed0eb359d1920f62a59f093bb90f7dd  .git/patches/master/series
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -124,7 +123,6 @@ f da39a3ee5e6b4b0d3255bfef95601890afd807
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 3516b9a25ed0eb359d1920f62a59f093bb90f7dd  .git/patches/master/series
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
Index: guilt-0.36/regression/t-035.out
===================================================================
--- guilt-0.36.orig/regression/t-035.out
+++ guilt-0.36/regression/t-035.out
@@ -43,7 +43,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -97,7 +96,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 51fcfcf16db2903f19ab4a4a3caacd297ea9f6cd  .git/patches/master/empty~
@@ -153,7 +151,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 51fcfcf16db2903f19ab4a4a3caacd297ea9f6cd  .git/patches/master/nonempty~
@@ -211,7 +208,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 51fcfcf16db2903f19ab4a4a3caacd297ea9f6cd  .git/patches/master/nonempty~
@@ -272,7 +268,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 51fcfcf16db2903f19ab4a4a3caacd297ea9f6cd  .git/patches/master/nonempty~
@@ -332,7 +327,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 51fcfcf16db2903f19ab4a4a3caacd297ea9f6cd  .git/patches/master/empty~
@@ -392,7 +386,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 51fcfcf16db2903f19ab4a4a3caacd297ea9f6cd  .git/patches/master/nonempty~
@@ -452,7 +445,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 51fcfcf16db2903f19ab4a4a3caacd297ea9f6cd  .git/patches/master/nonempty~
Index: guilt-0.36/regression/t-051.out
===================================================================
--- guilt-0.36.orig/regression/t-051.out
+++ guilt-0.36/regression/t-051.out
@@ -117,7 +117,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -231,7 +230,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -340,7 +338,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 4a83e5a312b6e25e333400a5ebdf19a297a7ecfc  .git/patches/master/series
@@ -454,7 +451,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 4a83e5a312b6e25e333400a5ebdf19a297a7ecfc  .git/patches/master/series
@@ -565,7 +561,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 4a83e5a312b6e25e333400a5ebdf19a297a7ecfc  .git/patches/master/series
@@ -689,7 +684,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 03416e9c8dd788d6cb1b318d74ff34f4d6150c44  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -798,7 +792,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 03416e9c8dd788d6cb1b318d74ff34f4d6150c44  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -914,7 +907,6 @@ All patches popped.
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
 f 03416e9c8dd788d6cb1b318d74ff34f4d6150c44  .git/patches/master/series
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
Index: guilt-0.36/regression/t-061.out
===================================================================
--- guilt-0.36.orig/regression/t-061.out
+++ guilt-0.36/regression/t-061.out
@@ -215,8 +215,6 @@ d4850419ccc1146c7169f500725ce504b9774ed0
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
-d .git/refs/patches/topic
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -263,8 +261,6 @@ d4850419ccc1146c7169f500725ce504b9774ed0
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
-d .git/refs/patches/topic
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -281,8 +277,6 @@ d .git/patches
 d .git/patches/master
 d .git/patches/topic
 d .git/refs/patches
-d .git/refs/patches/master
-d .git/refs/patches/topic
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/topic/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -310,8 +304,6 @@ d4850419ccc1146c7169f500725ce504b9774ed0
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
-d .git/refs/patches/topic
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -394,8 +386,6 @@ d4850419ccc1146c7169f500725ce504b9774ed0
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
-d .git/refs/patches/topic
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
Index: guilt-0.36/regression/t-062.out
===================================================================
--- guilt-0.36.orig/regression/t-062.out
+++ guilt-0.36/regression/t-062.out
@@ -181,8 +181,6 @@ d4850419ccc1146c7169f500725ce504b9774ed0
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
-d .git/refs/patches/topic
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -226,8 +224,6 @@ d4850419ccc1146c7169f500725ce504b9774ed0
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
-d .git/refs/patches/topic
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -244,8 +240,6 @@ d .git/patches
 d .git/patches/master
 d .git/patches/topic
 d .git/refs/patches
-d .git/refs/patches/master
-d .git/refs/patches/topic
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/topic/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
@@ -273,8 +267,6 @@ d4850419ccc1146c7169f500725ce504b9774ed0
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
-d .git/refs/patches/topic
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
@@ -354,8 +346,6 @@ d4850419ccc1146c7169f500725ce504b9774ed0
 d .git/patches
 d .git/patches/master
 d .git/refs/patches
-d .git/refs/patches/master
-d .git/refs/patches/topic
 f 22930c6d1f1938f298a4fca51c57e4b47171db21  .git/patches/master/mode
 f 413390f3906f16f30b054a4fb86c1e014b964504  .git/patches/master/remove
 f 9c18cc7abe6b87f18503714a80a677b4094eb457  .git/patches/master/add
--- guilt-0.36/regression/t-034.out.1	2018-01-25 08:27:41.417522219 +0000
+++ guilt-0.36/regression/t-034.out	2018-01-25 08:28:36.391120747 +0000
@@ -307,7 +307,7 @@
 Applying patch..can-have-embedded-single-slashes.patch
 Patch applied.
 % git log --decorate
-commit 434e07cacdd8e7eb4723e67cb2d100b3a4121a3a (HEAD, refs/patches/master/can-have-embedded-single-slashes.patch, guilt/master)
+commit 434e07cacdd8e7eb4723e67cb2d100b3a4121a3a (HEAD -> guilt/master, refs/patches/master/can-have-embedded-single-slashes.patch)
 Author: Author Name <author@email>
 Date:   Mon Jan 1 00:00:00 2007 +0000
 
